src/commons/utils/yaml_parser.py

```diff
@@ -122,6 +122,13 @@ def test_parser(yaml_file, number_of_nodes):
                         return False
                     data[size_type]["start"] = convert_to_bytes(data[size_type]["start"])
                     data[size_type]["end"] = convert_to_bytes(data[size_type]["end"])
+                elif isinstance(data[size_type], list):
+                    out = []
+                    for item in data[size_type]:
+                        out.append(convert_to_bytes(item))
```

Comment:
I didn't get this logic. @azaharkhan  can you please explain it?

Reply:
As mentioned in above comment, checking condition for list and used for loop for converting all the fixed object sizes from the list in to bytes.